fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/FallbackCursor.java

```diff
@@ -138,6 +137,18 @@ public boolean accept(@Nonnull RecordCursorVisitor visitor) {
         return visitor.visitLeave(this);
     }
 
+    private Throwable wrapException(final String msg, final Throwable ex) {
+        if ((ex.getCause() != null) && (ex.getCause() instanceof LoggableException)) {
+            // in the case of loggable exception, maintain the original exception to simplify exception handling across
+            // fallback and non-fallback executions
+            LoggableException loggableException = (LoggableException)(ex.getCause());
+            loggableException.addLogInfo("FallBackFailed", msg);
```

Comment:
Something like `fallback_failed` should probably be added to the `LogMessageKeys`. Or it may be sufficient to make this `fallback_failed` (as our convention for log keys is lowercase\_underscores rather than UpperCamel).

Reply:
Done (renamed to fallback_failed)